stdlib/2/builtins.pyi

```diff
@@ -66,18 +66,18 @@ class classmethod(object):  # Special, only valid as a decorator.
 
 class type(object):
     __bases__ = ...  # type: Tuple[type, ...]
-    __name__ = ...  # type: str
-    __module__ = ...  # type: str
+    __name__ = ...  # type: Text
```

Comment:
I don't think this is correct. `__name__` is always str in Python 2, not unicode.